taxcalc/records.py

```diff
@@ -25,31 +32,178 @@ class Records(object):
     constructor.
     Advancing years is done through a member function
     """
-    CUR_PATH = os.path.abspath(os.path.dirname(__file__))
-    WEIGHTS_FILENAME = "WEIGHTS.csv"
-    weights_path = os.path.join(CUR_PATH, WEIGHTS_FILENAME)
-    BLOWUP_FACTORS_FILENAME = "StageIFactors.csv"
-    blowup_factors_path = os.path.join(CUR_PATH, BLOWUP_FACTORS_FILENAME)
 
     @classmethod
     def from_file(cls, path, **kwargs):
         return cls(path, **kwargs)
 
-    def __init__(self,
-                 data="puf.csv",
-                 blowup_factors=blowup_factors_path,
-                 weights=weights_path,
-                 start_year=None,
+    def __init__(self, data=None, blowup_factors=blowup_factors_path,
+                 weights=weights_path, start_year=None, dims=None,
                  **kwargs):
 
-        self.read(data)
-        self.read_blowup(blowup_factors)
-        self.read_weights(weights)
+        self.BF = read_blowup(blowup_factors)
+        self.WT = read_weights(weights)
+
+        attr = ['AGIR1', 'DSI', 'EFI', 'EIC', 'ELECT', 'e04600',
```

Comment:
In particular, the list here:
https://github.com/mmessick/Tax-Calculator/blob/luca/taxcalc/records.py#L467
is the same list